### PR TITLE
fix(agents-api): added anthropic call in chat endpoint + typespec changed to include imageurl in ChatInput

### PR DESCRIPTION
**Pull request by Vedantsahai18**
> [!IMPORTANT]
> Integrates Anthropic API into chat endpoint, updates models for image content, and improves tool formatting and error handling.
> 
>   - **Behavior**:
>     - Added Anthropic API call in `chat.py` for tool types `computer_20241022`, `bash_20241022`, `text_editor_20241022`.
>     - Updated `request_anthropic()` to format messages and call Anthropic API.
>     - Modified tool formatting logic in `chat.py` for Anthropic tools.
>   - **Models**:
>     - Added `ChatMLAnthropicImageContentPart` and related models in `models.tsp` for Anthropic image content.
>     - Updated `ChatMLContentPart` alias to include `ChatMLAnthropicImageContentPart`.
>   - **Misc**:
>     - Updated `openapi-1.0.0.yaml` to reflect changes in `ChatInput` and related schemas.
>     - Added `Source` class in `Chat.py`, `Entries.py`, and `Tasks.py` for base64 image data.
>     - Improved error handling in `execute_integration.py` and `execute_system.py`.
>     - Updated Gunicorn configuration in `gunicorn_conf.py` for testing and debugging.
>     - Added `GEMINI_API_KEY` to `.env.example` and `docker-compose.yml`.
> 
> <sup>This description was created by </sup>[Ellipsis](https://www.ellipsis.dev?ref=julep-ai%2Fjulep&utm_source=github&utm_medium=referral)<sup> for bc0e0d4bf6b7bd5af064aa1fcc8ae90371dc00ac. It will automatically update as commits are pushed.</sup>

### CODE REVIEW

File: typespec/entries/models.tsp [hunk 1]
```diff
@@ -50,7 +50,25 @@ model ChatMLImageContentPart<T extends string = string> {
...
+}
+
+model ChatMLAnthropicInnerImageContentPart<T extends string = string> {
+    type: "image" = "image";
+    source: ChatMLAnthropicImageSource<T>;
+}
+
+/** Anthropic image content part */
+model ChatMLAnthropicImageContentPart<T extends string = string> {
+    tool_use_id: string;
```
Comment (HamadaSalhab) [inline typespec/entries/models.tsp:66]: - The `tool_use_id` should be optional to support the case where it's a normal completion request and not responding to a tool call.

- Likewise, the `type` should support other types such as `user`, `assistant`... etc.

**Review by HamadaSalhab — COMMENTED** [review]
Left you a comment on the typespec changes.

File: integrations-service/integrations/utils/integrations/remote_browser.py [hunk 2]
```diff
@@ -231,6 +231,16 @@ async def press_key(self, key_combination: str) -> RemoteBrowserOutput:
         """Press a key or key combination"""
         # Split combination into individual keys
         keys = key_combination.split("+")
+        keys = [
```
Comment (creatorrr) [inline integrations-service/integrations/utils/integrations/remote_browser.py:234]: Why is this needed?

Comment (Ahmad-mtos) [reply integrations-service/integrations/utils/integrations/remote_browser.py:234]: Because the key mappings of computer use are different from the ones used by playwright, would you rather we handle that in browser-control and not here?

Comment (creatorrr) [reply integrations-service/integrations/utils/integrations/remote_browser.py:234]: No this is fine

**Comment by creatorrr** [comment]
Lgtm. If Hamada’s comments are addressed then merge

File: typespec/entries/models.tsp (hunk 1 above)
Comment (creatorrr) [reply typespec/entries/models.tsp:66]: can you make theses changes and autogenerate @HamadaSalhab ?

File: agents-api/agents_api/routers/sessions/chat.py [hunk 3]
```diff
@@ -1,28 +1,151 @@
...
+                for tool_call in msg.get("tool_calls", [])
+            )
+            claude_message = {
+                "role": msg["role"],
+                "content": transformed_content,
+            }
+        elif msg["role"] == "user":
+            try:
+                transformed_content = json.loads(msg["content"])
+            except Exception:
```
[the remark this answers is not in the document]
Comment (Ahmad-mtos) [reply agents-api/agents_api/routers/sessions/chat.py:86]: I remember the exception wasn't being caught when I had ``json.JSONDecodeError``, will check it tomorrow

**Review by HamadaSalhab — APPROVED** [review]
Tested & confirmed. LGTM
